### PR TITLE
Feat/pvp  add under attack modal

### PR DESCRIPTION
**Pull request by Jcalcaldev**
This PR:

Changes the old generic alert for a custom modal to show players they're under attack.

### CODE REVIEW

File: frontend/src/components/smart/PvPUnderAttackModal.vue [hunk 1]
```diff
@@ -0,0 +1,196 @@
+<template>
+  <div @click="close" class="modalWrapper">
```
Comment (Chriserus) [inline frontend/src/components/smart/PvPUnderAttackModal.vue:2]: Why are you using a custom modal and not b-modal (like everywhere else)? Is this needed?

File: frontend/src/locales/fr.json [hunk 2]
```diff
@@ -584,7 +584,9 @@
   "pvp": {
     "lastDefeated": "Vous avez été vaincu pour la dernière fois par :",
     "tapAnywhere": "Appuyez n'importe où pour fermer",
-    "kicked": "Vous avez été expulsé de l'arène !"
+    "kicked": "Vous avez été expulsé de l'arène !",
```
Comment (Chriserus) [inline frontend/src/locales/fr.json:587]: Just so you know, you don't need to provide all the translations :D There's always fallback for english, and translations can be handled by translators. Nice addition nevertheless!

**Review by Chriserus — APPROVED** [review]

File: frontend/src/components/smart/PvPUnderAttackModal.vue (hunk 1 above)
Comment (Jcalcaldev) [reply frontend/src/components/smart/PvPUnderAttackModal.vue:2]: Not absolutely, we can write an issue to migrate our custom modals to b-modals if you wish so

File: frontend/src/locales/fr.json (hunk 2 above)
Comment (Jcalcaldev) [reply frontend/src/locales/fr.json:587]: Too late! Got most of the translations ready in another PR, should I delete them and leave them to the translators or leave them in? I would assume some of them might be wrong due to Google translate, but I don't know what's easier for them :o

Comment (Chriserus) [reply frontend/src/locales/fr.json:587]: Just leave it however you've done it :D Not a problem for them, just to assure you, you don't need to provide ALL translations, just english, that's why I was surprised you guys didn't want to do them on the spot. It's just adding a key value in en.json, the rest will be handled.

File: frontend/src/components/smart/PvPUnderAttackModal.vue (hunk 1 above)
Comment (Chriserus) [reply frontend/src/components/smart/PvPUnderAttackModal.vue:2]: Please do

**Pull request merged**